IDEAS/Buildings/Components/Window.mo

```diff
@@ -22,7 +22,8 @@ model Window "Multipane window"
       mats=glazing.mats,
       energyDynamics=if windowDynamicsType == IDEAS.Buildings.Components.Interfaces.WindowDynamicsType.Normal then energyDynamics else Modelica.Fluid.Types.Dynamics.SteadyState,
       dT_nom_air=5,
-      linIntCon=true));
+      linIntCon=true),
+      errorLowPerformance = errorLowPerformance);
```

Comment:
I don't think that we need this parameter here? You can directly set `multiLayer.errorLowPerformance = glazing.errorLowPerformance`